install/kubernetes/helm/istio/charts/servicegraph/templates/deployment.yaml

```diff
@@ -12,27 +12,26 @@ spec:
   template:
     metadata:
       labels:
-        app: {{ template "servicegraph.name" . }}
-        release: {{ .Release.Name }}
+        app: servicegraph
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
       containers:
-        - name: {{ .Chart.Name }}
-          image: "{{ .Values.global.servicegraph.repository }}:{{ .Values.global.servicegraph.tag }}"
+        - name: servicegraph
+          image: "{{ .Values.global.hub }}/servicegraph:{{ .Values.global.tag }}"
           imagePullPolicy: {{ .Values.imagePullPolicy }}
           ports:
             - containerPort: {{ .Values.service.internalPort }}
           args:
           - --prometheusAddr={{ .Values.prometheusAddr }}
-          livenessProbe:
-            httpGet:
-              path: /
-              port: {{ .Values.service.internalPort }}
-          readinessProbe:
-            httpGet:
-              path: /
-              port: {{ .Values.service.internalPort }}
+#          livenessProbe:
```

Comment:
commented out because it fails - 404. We need to add it back, but not sure what URL.